src/applications/vaos/sass/vaos.scss

```diff
@@ -1,7 +1,7 @@
 @import "~@fortawesome/fontawesome-free/scss/fontawesome";
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-process-list";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/color";
+@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities";
```

Comment:
hmmm this isn't doing what we'd expect it to: 
![image](https://github.com/user-attachments/assets/c121473f-6155-4355-b5b4-599bc85039b6)

It's getting the utility class names, but because the same names in Formation include `!important`, it continues to inherit from there instead. 

Somewhere in the vets-website css import architecture, the import for `/formation/sass/utilities/color` is overriding this. Probably in [the core.scss import](https://github.com/department-of-veterans-affairs/vets-website/blob/9b2f222cdfab2df86e6901c2b609772d36561390/src/platform/site-wide/sass/style.scss#L5) we've been talking about.